open-source.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Minimum Open Source requirements for Services
+description: The minimum bar for Open Source when working on Services, that then allows further progression.
+---
+
+# Minimum Open Source Requirements for Services
+
+Here we document the minimum bar for use of Open Source in services. When a service meets this minimum bar, it is then possible for contributors to further progress the service toward the complete Operate First methodology. This represents what we expect from any service claiming to be Open Source.
```

Comment:
Note it's a good practice to put a sentence alone on each line for easier to read diffs. This will be part of our docs standard for the project.

Reply:
```suggestion
Here we document the minimum requirements for use of Open Source in services. When a service meets this minimum requirement, it is then possible for contributors to further progress the service toward the complete Operate First methodology. This represents what we expect from any _service_ claiming to be Open Source.
```

Reply:
Split sentences ... and changed usage of word "bar" to "requirements" throughout.